.eslintrc

```diff
@@ -1,3 +1,22 @@
 extends: tlvince
 env:
   node: true
+
+rules:
+  # These rules have been modified in the update of eslint@^1.0.0.
+  # The configuration is overridden, because the upstream ruleset hasn't been upgraded to this eslint version.
```

Comment:
Thanks. I should update eslint-config-tlvince, though I may switch over to standard.